src/options.js

```diff
@@ -2,6 +2,8 @@
 
 const req = require('import-cwd')
 
+const { stripRequireStack } = require('./utils.js')
```

Comment:
Why is this needed ?  🙃

Reply:
Tests were failing because when require() failed, it was throwing an error that looked something like:

```
Failed to load module: 'whatever'
Require stack:
- ...
- ...
- ...
```

It seems the original code was not expecting that "require stack" stuff, so the tests were failing. I'm not sure if it's because I'm using a newer version of node or what, but I wrote a utility function to strip that stuff out. It's not really useful in the error message anyway, since the stack is deep in postcss-load-config.

Reply:
ahhh ok so 'require stack' refers to the Error Stacktrace (`err.stack`) in general? Never saw the `Require stack: ...` prefix before tbh, maybe cosmiconfig formats the `{Error}` message 🤷‍♂. Rename it to `stripStack` or maybe even `format` please

Reply:
😨 Wait a sec... I just saw this modifies the `{Error}` itself and doesn't strip off the satck for tests only 🤦‍♂. We need to be careful here as this might introduce a breaking change. Which test where failing in particular?

Personally agreeing with the `stack` being rather uninteresting here.

**src/Error.js**
```js
class ConfigError extends Error {
   constructor (message) {
     super()
     
     this.name = 'ConfigError'
     this.message = message
     // this.stack = false
       
     Error.captureStackTrace(this, this.constructor)
   }
}

module.exports = ConfigError
```

Reply:
The "Require stack" appears in the error message. It's not actually modifying the error, it's just taking the message from the error and stripping out the "require stack" stuff.

Pretty much every test in Errors.test.js was failing without this change, though, it's weird... I just tried it on a different laptop to get you some example test output and it worked fine, so...

¯\\\_(ツ)\_/¯

I'll revert this part of my PR since I can no longer reproduce the issue...